core/embed/rust/build.rs

```diff
@@ -158,6 +159,8 @@ fn prepare_bindings() -> bindgen::Builder {
             .map(|s| format!("-I{}", s.trim()));
 
         bindings = bindings.clang_args(include_args);
+    } else {
+        clang_args.push("fno-short-enums");
```

Comment:
is the missing dash a typo or intentional?

Reply:
typo https://github.com/trezor/trezor-firmware/commit/e9407a75106124c64d6954cfcaedc7aabc989a5d